### PR TITLE
add nightly builds for iOS

### PR DESCRIPTION
**Pull request by v-almonacid**
In order to have a separate app with a different icon I needed to add:
- A new target (emurgo-nightly)
- A new scheme (emurgo-nightly)

The new scheme is set to use the new app id defined in the new target. So now nightly builds can also be installed in parallel on iOS:

Screenshot 2020-03-09 at 20 05 44

### CODE REVIEW

File: ios/Staging-Info.plist [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<string>47</string>
```
Comment (SebastienGllmt) [inline ios/Staging-Info.plist:24]: Why change this to a hard-coded number?

Comment (v-almonacid) [reply ios/Staging-Info.plist:24]: I haven't touched that field, this was done by Xcode internally. Actually I only modified two files directly: `Fastfile` and fastlane's README. All other modifications are the result of creating and configuring a new *scheme* and a new *target* in Xcode.

Comment (v-almonacid) [reply ios/Staging-Info.plist:24]: I was wrong, this was done automatically by fastlane when a new build is pushed. I'm setting up all this again in a new branch to get a cleaner result.

**Comment by v-almonacid** [comment]
Deprecated in favor of https://github.com/Emurgo/yoroi-mobile/pull/808

**Pull request closed without merging**